### ISSUE TITLE
Question about hyperparameter tuning

### ISSUE THREAD

**Issue by PengxinWang** [ilnehc96/HKU-DASC7606-A1#4]
There are a lot of hyperparameters in the retinanet model, such as $\alpha$, $\beta$ in the focal loss, model depth, learning rate, e.t.c. However, it takes roughly 5 minutes to train a epoch and about 4 hours to train 40 epochs. It's really expensive to do it by trial and error and we only have 100 hours HKU Gpu Farm quota. So how to tune these hyperparameters in methods like grid search? And how to do analysis based on limited experiments.

It'd be appreciated if you can give some suggestions or reference!
